frontend/src/store/actions/product.js

```diff
@@ -85,3 +90,32 @@ export const createExistingProduct = form => (dispatch, getState) => {
       console.log(error);
     });
 };
+
+export const updateProduct = form => (dispatch, getState) => {
+  const config = {
+    headers: {
+      "Content-Type": "application/json"
+    }
+  };
+
+  const body = {
+    action: "updateProduct",
+    data: {
+      customerId: form["customerId"],
+      productId: form["productId"],
+      name: form["name"]
+    }
+  };
+
+  axios
+    .post("http://localhost/mmcSupport/backend/", body, config)
```

Comment:
die url http://localhost/mmcSupport/backend/ naar een losse variable zetten het liefst in een andere file en die je dan kan gebruiken bij elke call.